test/cell_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule GameOfLife.CellTest do
+  use ExUnit.Case, async: true
+
+  test "A Cell can stay or come alive" do
+    location = {1,1}
+    dead_cell = %{location => :dead}
+    living_cell = %{location => :alive}
+    assert GameOfLife.Cell.live(dead_cell,location) == %{{1,1} => :alive}
+    assert GameOfLife.Cell.live(living_cell,location) == %{{1,1} => :alive}
+  end
+
+  test "A Cell can stay or be dead" do
+    location = {1,1}
+    dead_cell = %{location => :dead}
+    living_cell = %{location => :alive}
+    assert GameOfLife.Cell.die(dead_cell,location) == %{{1,1} => :dead}
+    assert GameOfLife.Cell.die(living_cell,location) == %{{1,1} => :dead}
+  end
+
+  test "A living Cell responds true to is_alive?" do
```

Comment:
Can you nest your tests in erlang, like how ruby does a describe? That would be much nicer.